react-search-component/my-app/src/SearchableList Components/SearchBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEvent } from 'react';
+
+type SearchBarProps = {
+  onSearch: (searchText: string) => void;
+};
+
+export default function SearchBar({ onSearch }: SearchBarProps) {
+  function handleOnChange(event: ChangeEvent<HTMLInputElement>) {
+    onSearch(event.target.value);
+  }
+
+  return (
+    <div>
+      <label>
+        <input
+          onChange={handleOnChange}
+          className="m-2 p-1 border border-solid border-black rounded w-1/4"
```

Comment:
I enjoy using Tailwind. It's impressive you've already figured out how to make use of it!

Reply:
Thanks! It made me love CSS a lot more since I can see everything all in one page haha